indev/libinput.c

```diff
@@ -410,39 +493,55 @@ static void reset_scanned_devices(void) {
 static void read_pointer(libinput_drv_state_t *state, struct libinput_event *event) {
   struct libinput_event_touch *touch_event = NULL;
   struct libinput_event_pointer *pointer_event = NULL;
+  libinput_lv_event_t *evt = NULL;
   enum libinput_event_type type = libinput_event_get_type(event);
 
+  /* We only care about these events */
+  if (type != LIBINPUT_EVENT_TOUCH_MOTION &&
+      type != LIBINPUT_EVENT_TOUCH_DOWN &&
+      type != LIBINPUT_EVENT_TOUCH_UP &&
+      type != LIBINPUT_EVENT_POINTER_MOTION &&
+      type != LIBINPUT_EVENT_POINTER_BUTTON)
+    return;
+
   /* We need to read unrotated display dimensions directly from the driver because libinput won't account
    * for any rotation inside of LVGL */
   lv_disp_drv_t *drv = lv_disp_get_default()->driver;
 
+  /* ignore more than 2 fingers as it will only confuse LVGL */
+  touch_event = libinput_event_get_touch_event(event);
```

Comment:
This triggers a warning on the console when the event is one of the `LIBINPUT_EVENT_POINTER_*` types. So for mice and trackpads this prints on every cursor motion or key press.

Reply:
Proposal how to fix this: https://github.com/calebccff/lv_drivers/pull/1